### PR TITLE
Add CustomEvent polyfill for browser only

### PR DESCRIPTION
**Pull request by raejin**
Adding CustomEvent support for IEs.

### CODE REVIEW

File: browser-only.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ if (typeof window !== 'undefined') {
   // Polyfill window.matchMedia (primarily for IE9)
   require('matchmedia-polyfill');
   require('matchmedia-polyfill/matchMedia.addListener');
+
+  // Polyfill CustomEvent
+  require('custom-event-polyfill');
```
Comment (ljharb) [inline browser-only.js]: Any reason to prefer this one, vs https://www.npmjs.com/package/custom-event? I have no preference, just want to make sure we're using the best/most maintained one.

Comment (raejin) [reply browser-only.js]: @ljharb I think I got fooled by SEO, was not aware of this. I'll update accordingly. Thanks for catching this.

**Comment by raejin** [comment]
@ljharb Confirmed working in IE9, IE10, and IE11.

**Review by ljharb — APPROVED** [review]
LGTM

**Review by ljharb — CHANGES_REQUESTED** [review]
Oh, actually, let's add this to the readme too :-)

**Comment by raejin** [comment]
@ljharb updated!

File: README.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ import 'airbnb-browser-shims';
  - `TouchList[Symbol.iterator]` - when `Symbol` and `TouchList` are both present
  - [element-closest](https://npmjs.com/element-closest) - Element.prototype.closest polyfill (only in browsers)
  - [smoothscroll-polyfill](https://www.npmjs.com/package/smoothscroll-polyfill) - Scroll behavior specification polyfill (only in browsers)
+ - [custom-event](https://www.npmjs.com/package/custom-event) - CustomEvent polyfill (only in browsers)
```
Comment (ljharb) [inline README.md]: it's not only in browsers tho, is it? it looks like it's doing it in node too

**Review by ljharb — APPROVED** [review]

**Comment by ljharb** [comment]
Ended up reverting this by rewinding; we don't need it long term.

**Pull request merged**